### DISCUSSION TITLE
Is there any 2.2.0 binaries for macOS?

### DISCUSSION THREAD

**Discussion by maikol-solis** [lem-project/lem#1841] [Q&A]
Hi. 

I would like to start testing LEM on macOS, but I couldn't find any binaries here (https://github.com/lem-project/lem/releases/tag/v2.2.0) or in brew.sh. 

Any assistance is appreciated it. 

Best.

**Comment by sakurawald** [comment]
Hello, I am using the `lem editor v2.2.0 sdl2 version` in `linux`.
I only can find the `v2.1.0` version in `archlinux user repository`, so i clone the source, and follow the guide to compile and install the latest version of lem for my self.

I don't know if there is the latest binary version for macos. 
However, you can follow the guide to compile it from source: https://lem-project.github.io/installation/sdl2/macos/
